libs/nghttp3/Makefile

```diff
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nghttp3
-PKG_VERSION:=1.1.0
-PKG_RELEASE:=1
+PKG_VERSION:=1.2.0
+PKG_RELEASE:=r1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/ngtcp2/nghttp3/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=b3ffb23a90442a0eafe8bfbefbc8b4ffb5179d68a7c0b8a416a34cf04b28d7c5
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://github.com/ngtcp2/$(PKG_NAME)/releases/download/v$(PKG_VERSION)/
```

Comment:
Please dont reuse PKG_NAME variable here. You can find it in some pull request, why it is not a good idea.

If you will copy it to the browser, it will not work, so keep it as it was in the previous, so:
https://github.com/ngtcp2/nghttp3/releases/download/v$(PKG_VERSION)

There is nothing wrong with that.

You are also changing it from codeload to GitHub. It is not mentioned in the commit description.

Reply:
NACK: https://github.com/search?q=repo%3Aopenwrt%2Fpackages%20%24(PKG_NAME)%2Freleases&type=code

Reply:
@stangri I am regret to say, but it looks like you are doing what you want. This does not look good. How many URL based on your findings are using PKG_NAME in the PKG_SOURCE_URL? Just a few packages. Not all. Why?

Sorry, but you posted almost random URL and said NACK. You havent do anything and simply merged, because you can. This is not how the things works.